src/views/TaskView.vue

```diff
@@ -43,7 +45,15 @@ const loadTasks = () => {
       console.error('Error loading tasks:', error);
     });
 };
-
+const filterMineTasks = () => {
+  const filteredTasks = tasks.value.filter((task) => task.owner === props.owner);
+  return filteredTasks;
+};
+const filterOpenTasks = () => {
+  const filteredTasks = tasks.value.filter((task) => task.status.toString() === 'OPEN');
+  console.log('filtered tasks', filteredTasks);
+  return filteredTasks;
+};
```

Comment:
While the current frontend filtering works, it would be more efficient to implement filtering directly in the backend API. This approach would allow the frontend to request only the necessary tasks, reducing the amount of data transferred over the network and improving performance, especially as the dataset grows. The backend could support query parameters like owner and status (e.g., /tasks?owner=<owner>&status=OPEN). This would also simplify the frontend logic by removing the need for local filtering